web-registry/src/pages/Ecocredits.tsx

```diff
@@ -0,0 +1,257 @@
+import React, { useEffect, useState } from 'react';
+import { format } from 'date-fns';
+import {
+  Table,
+  TableBody,
+  TableHead,
+  TableRow,
+  Box,
+  TableFooter,
+} from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import {
+  StyledTableCell,
+  StyledTableContainer,
+  StyledTableRow,
+  StyledTableSortLabel,
+} from 'web-components/lib/components/table';
+import Section from 'web-components/lib/components/section';
+import { TableActionButtons } from 'web-components/lib/components/buttons/TableActionButtons';
+import {
+  getComparator,
+  Order,
+  stableSort,
+} from 'web-components/lib/components/table/sort';
+import { getAccontEcocreditsForBatch, getBatches } from '../lib/ledger-rest';
+import { useTablePagination } from 'web-components/lib/components/table/useTablePagination';
+import { ledgerRestUri } from '../ledger';
+import { truncateWalletAddress } from '../lib/wallet';
+import { BatchRowData, EcocreditAccountBalance } from '../types/ledger';
+import { getAccountUrl } from '../lib/block-explorer';
+
+const useStyles = makeStyles(theme => ({
+  borderLeft: {
+    // absolutely position border to get around MUI style quirks
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    bottom: 0,
+    borderLeft: `1px solid ${theme.palette.info.light}`,
+  },
+  greyText: {
+    color: theme.palette.info.main,
+  },
+}));
+
+const ROWS_PER_PAGE_OPTIONS = [5, 10];
+
+interface TableCredits extends BatchRowData, EcocreditAccountBalance {}
+
+export const Ecocredits: React.FC = () => {
+  // const walletContext = useWallet();
+  // const wallet = walletContext.wallet?.address;
+  // TODO hard coded for now - the following should work for a user's data
+  const wallet = 'regen1m5fecarvw0ltx2yvvru0kl4un03d3uca2kxggj';
```

Comment:
hard coded to show functionality - should update before we go live with this feature

Reply:
Maybe we can transfer some ecocredits to some of our test wallets on Hambach? I don't know how we can test this otherwise. 

cc @blushi

Reply:
I guess we can assume it will work without needing to have ecocredits in our wallets. But it is something we will want eventually.

Reply:
since Hambach is down and doesn't produce any blocks for now, we'll need to switch to Redwood if we want to transfer some ecocredits and test this with our own address
We'll need this anyway for implementing transfer/retire if Hambach is not upgraded by then, so please @mhagel @haveanicedavid could you share your wallet addresses so I can send you some ecocredits on Redwood eventually?